### DISCUSSION TITLE
GTC'23 Spring CUTLASS talks

### DISCUSSION THREAD

**Discussion by hwu36** [NVIDIA/cutlass#882] [Show and tell]
There are going to be 4 talks this time.

- March 22, [Developing Optimal CUDA Kernels on Hopper Tensor Cores](https://register.nvidia.com/flow/nvidia/gtcspring2023/attendeeportal/page/sessioncatalog/session/1666444617375001kQdj) from CUTLASS team
- March 20, [Accelerating Large Language Models via Low-Bit Quantization](https://register.nvidia.com/flow/nvidia/gtcspring2023/attendeeportal/page/sessioncatalog/session/1666226207768001N4Fe) together with Microsoft
- March 22, [CUTLASS 极致性能优化探索及在阿里巴巴推荐系统中的应用](https://register.nvidia.com/flow/nvidia/gtcspring2023/attendeeportal/page/sessioncatalog/session/1666343524355001XrO5) from Alibaba
- March 22, [GPU 加速大规模向量索引构建](https://register.nvidia.com/flow/nvidia/gtcspring2023/attendeeportal/page/sessioncatalog/session/1666593153592001eePE) from Tencent

The conference is free and virtual.  Welcome to attend.

**Comment by soraros** [comment]
Thanks a lot for sharing!

Though I think it would help if this discussion can be moved to the category [Show and tell](https://github.com/NVIDIA/cutlass/discussions/categories/show-and-tell) (or Community news) for better discoverability and easy reference.
